test/data/test_multi_nested_tensor.py

```diff
@@ -1,10 +1,22 @@
 import random
+from typing import List
 
 import torch
+from torch import Tensor
 
 from torch_frame.data import MultiNestedTensor
 
 
+def assert_equal(tensor_mat: List[List[Tensor]],
+                 multi_nested_tensor: MultiNestedTensor):
+    assert len(tensor_mat) == multi_nested_tensor.num_rows
+    assert len(tensor_mat[0]) == multi_nested_tensor.num_cols
```

Comment:
Can we also expose `shape`? I think `num_rows`, `num_cols` terminology for a tensor is quite confusing.